test/functional/apps/visualize/_area_chart.js

```diff
@@ -106,12 +106,40 @@ bdd.describe('visualize app', function describeIndexTests() {
     });
 
     bdd.it('should show correct chart, take screenshot', function () {
-      const xAxisLabels = [ '2015-09-20 00:00', '2015-09-21 00:00',
-        '2015-09-22 00:00', '2015-09-23 00:00'
+      const xAxisLabels = [
```

Comment:
Yep, the visualize data set include an entire .kibana index so the update to the config doc needs to happen second